.circleci/deploy-python-2.sh

```diff
@@ -1,11 +1,14 @@
 #!/bin/sh
 
 set -e
-if ! git rev-parse `python setup.py --version` 2>/dev/null ; then
+
+if ./detect-functional-changes.sh
+then
     git tag `python setup.py --version`
     git push --tags  # update the repository version
     python setup.py bdist_wheel  # build this package in the dist directory
     twine upload dist/* --username $PYPI_USERNAME --password $PYPI_PASSWORD  # publish
+    ssh -o StrictHostKeyChecking=no deploy-api@fr.openfisca.org  # Deploy the OpenFisca-France public API
```

Comment:
I don't understand why this is in the country template 🤔

Reply:
Excessive copy pasting